etc/PackageConfigurations/eos_package_description.json

```diff
@@ -49,13 +49,33 @@
 
         {"src":"Assets/Plugins/Source/Editor/*.cs", "dest": "Editor/"},
         {"src":"Assets/Plugins/Source/Editor/*.cs.meta", "dest": "Editor/"},
+        {"src":"Assets/Plugins/Source/Editor/Controls.meta", "dest":"Editor/"},
+        {"src":"Assets/Plugins/Source/Editor/Controls/*.cs", "dest":"Editor/Controls/"},
+        {"src":"Assets/Plugins/Source/Editor/Controls/*.cs.meta", "dest":"Editor/Controls/"},
         
         {"src":"Assets/Plugins/Source/Editor/Utility.meta", "dest": "Editor/"},
         {"src":"Assets/Plugins/Source/Editor/Utility/com.playeveryware.eos-Editor.steam.utility.asmdef", "dest": "Editor/Utility/"},
```

Comment:
The steam version button will depend on this to work. 
I can make a separate PR to explain.
For the purpose of this PR, it shouldn't matter.